### DISCUSSION TITLE
Lavalink node is not connected

### DISCUSSION THREAD

**Discussion by FaizHarofi** [lavalink-devs/Lavalink#1051] [Q&A]
My Lavalink node is not connected, How i can fix it?
![Screenshot 2024-06-02 165314](https://github.com/lavalink-devs/Lavalink/assets/110177729/84d2d6a6-2363-493f-9a72-11839bf42f2d)

**Comment by japlic** [comment]
Hi can u send code

**Reply by samuelcollins1** [reply]
Dudes so desperate for a play command he's going through github issues to ask for their code 💀

Context: Spent literally hours asking for code for a play command in LLPY server instead of reading the docs then asked why when they threw the example cog into their code why didn't it work.

**Reply by imanimtiyaz20** [reply]
> Dudes so desperate for a play command he's going through github issues to ask for their code 💀
> 
> Context: Spent literally hours asking for code for a play command in LLPY server instead of reading the docs then asked why when they threw the example cog into their code why didn't it work.

To not waste that guy time, the code @IzzNoLimit use is from https://github.com/wtfnotavailable/Discord-MusicBot and written in javascript, not python

**Reply by japlic** [reply]
lmao its not true

**Reply by FaizHarofi** [reply]
Thank you all, I finally got it working with my brain
